modules/serverless-kafka-sasl-source.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sasl-source_{context}"]
+= Configuring SASL authentication for Kafka sources
+
+_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed.
```

Comment:
```suggestion
_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. When using the SASL authentication on a cluster, users must provide credentials to the Knative for communicating with the Kafka cluster; otherwise, events cannot produce or be consumed.
```

Reply:
I think the current version is better because it feels more "active" - also, this content is reused across a few different modules, so I'd prefer to keep it consistent